community/assessments/projects/wasmedge/joint-assessment.md

```diff
@@ -0,0 +1,71 @@
+# WasmEdge TAG-Security Security Assessment
+<!-- cSpell:ignore Ying Shen Hsieh ibmibmibm hydai wasmedge Constanze Röedig entlein Camila Vilarinho camilaavilarinho victorjunlu Meng WASI crun userbase Uptane wasi-->
+
+Completed: 01 07 2025
+
+Security reviewers: Justin Cappos @JustinCappos, Marina Moore @mnm678, Constanze Röedig @entlein, Camila Vilarinho @camilaavilarinho, and Victor Lu @victorjunlu
+
+Project security lead: Hung-Ying Tai @hydai
+
+Project team: Hung-Ying Tai @hydai, Meng-Han Lee @dm4
+
+* Source code: <https://github.com/WasmEdge/WasmEdge>
+* Web site: <https://wasmedge.org/>
+
+## Background
+
+WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime designed for cloud-native edge computing environments.
+As the fastest WebAssembly VM currently available, WasmEdge provides a secure sandboxed environment for running untrusted or third-party code in edge computing, serverless, and IoT scenarios.
+The project addresses the critical need for secure code execution with minimal resource footprint, enabling safe plugin architectures and cross-platform compatibility across diverse deployment environments.
+
+### Maturity
+
+* Modern web applications: Isomorphic and Jamstack applications with server-side WebAssembly execution
+* Edge computing: Microservices on edge cloud, IoT and embedded device applications
+* Serverless computing: Serverless SaaS APIs and Function-as-a-Service platforms
+* Smart contracts: Secure execution environment for blockchain applications
+* Machine learning at the edge: WASI-NN support for ML inference with frameworks like OpenVINO and PyTorch
+* Container ecosystem integration: Kubernetes WebAssembly runtime, Docker compatibility, CRI-O & crun support
+* Standards compliance: Full WebAssembly specification and WASI implementation
+
+## Summary
+
+**Design**: WasmEdge implements WebAssembly's security model with complete sandboxing, resource isolation, and controlled system access through WASI interfaces. The architecture includes validated bytecode loading, semantic verification, and fine-grained resource control for filesystem and memory access.
```

Comment:
I think you may have added this and don't think we really discussed this part of the text in the meeting.  A lot of this feels more like marketing speech.  If you want to discuss a way to describe some of this in the joint assessment document, we can talk amongst the reviewers and see what should make it and how to frame it differently.   However, to do this, please discuss with us in the slack channel instead of just making the edit.